Troubleshooting.md

```diff
@@ -0,0 +1,84 @@
+# Troubleshooting
+
+This document will help troubleshoot the common debugging errors. If you still cannot solve your problems, please [submit an issue](https://github.com/Microsoft/vscode-java-debug/issues).
+
+## Java Language Support extension fails to start.
+Java Debugger is leveraging `"Language Support for Java(TM) by Red Hat"` extension for source mapping and project support. If Java Language Support extension fails to start, Java Debugger may not work as expected. Here is a simple way to check the Language Server is started. Open a Java or pom.xml file, and then check the status icon at the right corner of VS Code status bar is thumbs up or not.
+
+ ![ status indicator ](https://raw.githubusercontent.com/redhat-developer/vscode-java/master/images/statusMarker.png).
+
+### Try:
+1. If you get the error `"The JAVA_HOME environment variable points to a missing folder"` or `"Java runtime could not be located"`, please make sure the JAVA_HOME and bin path point to a valid JDK home directory. Otherwise, ignore this step.

```

Comment:
please make sure that the environment variable `JAVA_HOME` points to a valid JDK.

Reply:
`"The JAVA_HOME environment variable points to a missing folder"` -> *"The JAVA_HOME environment variable points to a missing folder"*

Reply:
`"Java runtime could not be located"` -> *"Java runtime could not be located"*